code/drasil-example/Drasil/PDController/TModel.hs

```diff
@@ -17,6 +17,7 @@ import Utils.Drasil
 
 import Data.Drasil.Citations(laplaceWiki)
 
+
```

Comment:
why the extra blank line?

Also, this PR should delete `symnegInf` and `sumposInf` from PDController at the same time.

A separate PR can probably be made for re-using these in other examples.

Reply:
What do you mean by 'format' here?  Take a look at what is already in `drasil-data` as a guide for how it is currently done. I think `Pi` already exists, so do the same for the infinities, for example.

Reply:
oh okay okay, i see Pi, i'll write them in as such then.
by "format" i was asking what the line should look like. initially i just wrote them in the same way they were put in the PDController/concepts file